securedrop/source_templates/lookup.html

```diff
@@ -17,11 +17,11 @@
   {% endif %}
 </div>
 
-<h1 class="headline">{{ gettext('Submit Materials') }}</h1>
+<h1 class="headline">{{ gettext('Submit Materials or Messages') }}</h1>
 <p class="explanation">{{ gettext('If you are already familiar with GPG, you can optionally encrypt your files and messages with our <a href="{url}" class="text-link">public key</a> before submission. Files are encrypted as they are received by SecureDrop.').format(url=url_for('info.download_journalist_pubkey')) }}
 {{ gettext('<a href="{url}" class="text-link">Learn more</a>.').format(url=url_for('info.why_download_journalist_pubkey')) }}</p>
 
-<p class="explanation">{{ gettext('You can send a file, a message, or both.') }}</p>
+<p class="explanation">{{ gettext('You can submit any kind of file (up to 500 MB), a message, or both.') }}</p>
```

Comment:
The **(up to 500 MB)** part is redundant with what is displayed right under the upload icon.

Reply:
@dachary We can take this as an explicit message :)

Reply:
If it was your intention to duplicate this information, I think it is worth explaining how it helps in the commit message to clarify this is not by accident.

Reply:
I'm with @dachary. I think it's unnecessary to say it twice.